tests/foreman/cli/test_activationkey.py

```diff
@@ -619,17 +620,19 @@ def test_positive_usage_limit(module_org, module_location, target_sat):
             'lifecycle-environment-id': env['id'],
             'content-view': new_cv['name'],
             'organization-id': module_org.id,
-            'max-hosts': '1',
+            'max-hosts': max_hosts,
         }
     )
-    with Broker(nick='rhel7', host_class=ContentHost, _count=2) as clients:
+    with Broker(nick='rhel8', host_class=ContentHost, _count=2) as clients:
         vm1, vm2 = clients
         vm1.register(module_org, module_location, new_ak['name'], target_sat)
         assert vm1.subscribed
         result = vm2.register(module_org, module_location, new_ak['name'], target_sat)
         assert not vm2.subscribed
-        assert result.status == 70
-        assert len(result.stderr) > 0
+        assert result.status, 'Second registration was expected to fail'
+        assert (
+            f"Max Hosts ({max_hosts}) reached for activation key '{new_ak.name}'" in result.stderr
```

Comment:
Maintain consistency of string format in test module
```suggestion
            f'Max Hosts ({max_hosts}) reached for activation key "{new_ak.name}"' in result.stderr
```

Reply:
NACK, this would fail since there are single quotes used in the stderr, so I would need to escape them and it looks even worse.
```
Max Hosts (1) reached for activation key 'loremipsum'
```